application/config/constants.php

```diff
@@ -36,6 +36,51 @@
 define('FOPEN_WRITE_CREATE_STRICT',				'xb');
 define('FOPEN_READ_WRITE_CREATE_STRICT',		'x+b');
 
+/*
+ * ------------------------------------------------------------------------
+ * Province Settings
+ * ------------------------------------------------------------------------
+ *
+ * This array is for the dropdown-menu in the sign up form.
+ *
+ */
+
+$GLOBALS['PROVINCES'] = array(
```

Comment:
We can just use `$PROVINCES` here. $GLOBALS is only needed to access it in a different scope. [1]

[1] http://php.net/manual/en/language.variables.scope.php